README.md

```diff
@@ -68,8 +68,15 @@ underscores is hard. So for example, `add_repo` can be typed as `addrepo`.
                                 name given in the second parameter. The location
                                 of the repo will be asked for and then added to
                                 your list of repos.
+* `repo_add_directory` or `rad` - Searches all subdirectories of the second parameter
+                                  and automatically adds each to the list of repos. This
+                                  command uses the relative path between the second parameter 
+                                  and the repo as the repo name. The second parameter can be
```

Comment:
I'd like to change this to have it name the repo as just the name of the directory with the .git directory in it. For instance, if our second parameter is a directory named "code" and we have a directory that is a repo inside of that called "website", the repo name in alfred would just be "website."